src/components/post/index.js

```diff
@@ -62,6 +62,79 @@ const Post = ({
     }
   };
 
+  let likeIcon = null;
+  if (isLiked) {
+    likeIcon = <LikeRed />;
+  } else {
+    likeIcon = <Like />;
+  }
+
+    const showCommentModal = () => {
+        setModal('Add a comment...', 
+        <AddCommentModal 
+            setTriggerUpdate = {setTriggerUpdate}
+            id={id}
+            currentUserName={currentUserName}
+            currentUserInitials={currentUserInitials}
+            setUpdateComments={setUpdateComments}
+        />)
+        openModal()
+        }
+
+    const findComments = async () => {
+          const tempArray = [];
+          const res = await get(`posts/${id}/comments`);
+          const tempComments = res.data.comments;
+          await Promise.all(
+            tempComments.map(async (comment) => {
+              const userId = comment.userId;
+              const name = `${comment.user.profile.firstName} ${comment.user.profile.lastName}`;
+              const initials = `${comment.user.profile.firstName?.[0]}${comment.user.profile.lastName?.[0]}`;
+              const newComment = {
+                id: comment.id,
+                authorId: comment.userId,
+                name: name,
+                initials: initials,
+                text: comment.content,
+              };
+              tempArray.push(newComment);
+            })
+          );
+          const sortedArray = tempArray.sort((a,b) => a.id - b.id)
```

Comment:
What's the reasoning to sort by id - is timestamp data included that you can sort by?

Reply:
Timestamp data is included, but the array would still need sorting, not sorting it was resulting in the display order not being correct, and both sort by timestamp and by ID result in the same outcome and I chose ID.

Reply:
In general ids are just identifiers, they shouldn't be used to time sort data. It probably would be ok for a while until someone deleted a post and then inserted a post with specific id etc, ruining the world :)

If sorting by time, sort by time.

Reply:
PS: for a descending sort, just swap values 
```js
(a,b) => b.id - a.id
```